jake/day1/solutions/part2-readline.mjs

```diff
@@ -0,0 +1,54 @@
+console.time('runTime');
+import { once } from 'events';
+import { createReadStream } from 'fs';
+import * as readline from 'node:readline/promises';
+
+// Track X number of elfs at the top
+const FIND_TOP_X = 3;
+
+(async function () {
+  const rl = readline.createInterface({
+    input: createReadStream('../input/calories.txt'),
+    crlfDelay: Infinity, // Make "\r\n" a single newline
+  });
+
+  const topElves = Array(FIND_TOP_X).fill(0);
```

Comment:
I thought about doing this.  Instead of keeping an array of all elves to just keep the top N and insert in order each time.  🧠